scripts/python-skywater-pdk/skywater_pdk/liberty.py

```diff
@@ -38,6 +38,20 @@
 debug = False
 
 
+def msg_debug(*args, **kwargs):
```

Comment:
Could we use the standard Python logging interface here?

Reply:
Possibly ... but the classic stuff I use in web apps :
```
l = logging.getLogger(__name__)
l.debug('XXX')
```

Doesn't do anything here. Looks like you have to setup the whole logging infrastructure which is usually done by the framework but obviously not here, so that seems rather overkill to print a few messages. (also I have no clue how it works :sweat_smile: )

Reply:
I think we just need to add `logging.basicConfig(filename='example.log',level=logging.DEBUG)` to the main function?

See example below;
```python
import logging
import mylib

def main():
    logging.basicConfig(filename='myapp.log', level=logging.INFO)
    logging.info('Started')
    mylib.do_something()
    logging.info('Finished')

if __name__ == '__main__':
    main()
```

Reply:
It seems that the patch also improve on the error message themselves (adding `[prefix]`) in addition to moving stuff to stderr.

Given that the file still need to container some some vanilla print statement (in order to gather the list of deps as pointed by @smunaut), maybe we could simply move the other print to stderr by passing `file=sys.stderr` as a kwarg to the `print()` and get rid of the `msg_` functions?

That would make the patch simpler and leave more space to explore a `logging` refactoring as @mithro suggested.